chapters/CRUD-Operations.md

```diff
@@ -186,13 +186,13 @@ returns a `initialDocument`.
 
 ###### Deterministically Generate Beacon Services
 
-This algorithm deterministically generates three Beacons from the single
+This algorithm deterministically generates three ::Beacons:: from the single
 `keyBytes` value used to generate the deterministic **did:btc1**, one for each
 of the following three Bitcoin address types for the Bitcoin `network` specified
 by the DID: Pay-to-Public-Key-Hash (P2PKH), Pay-to-Witness-Public-Key-Hash (P2WPKH),
-and Pay-to-Taproot (P2TR). Spends from these three addresses can be produced only
+and Pay-to-::Taproot:: (P2TR). Spends from these three addresses can be produced only
```

Comment:
```suggestion
and Pay-to-Taproot (P2TR). Spends from these three addresses can be produced only
```

I do not think we should link to terminology here. Confusing as in the middle of the word